demos/crossroad_camera_demo/main.cpp

```diff
@@ -253,17 +253,22 @@ struct PersonAttribsDetection : BaseDetection {
     std::string outputNameForAttributes;
     std::string outputNameForTopColorPoint;
     std::string outputNameForBottomColorPoint;
+    bool hasTopBottomColor;
```

Comment:
I suggest checking `outputNameForTopColorPoint` or `outputNameForBottomColorPoint` emptiness instead of introducing `hasTopBottomColor`.

Reply:
IMHO it is not a good decision, since we will have two indirect conditions instead of one direct condition.

Everywhere we will have to check both of `outputNameForTopColorPoint` and `outputNameForBottomColorPoint`, since otherwise there will be not clear why we check one of them and do not check another.

In my opinion it is better to have a direct flag  with a clear name pointing if there are top/bottom point outputs or not.

Reply:
Ok